Azure/azure-cli
# {IOT} Iot central track 2 migration

**Description**
Updated IoT Central module from track 1 to track 2. Also, ran tests in live mode
**Testing Guide**
Ran the following  commands
o	“azdev style iot”
o	“azdev linter iot”
o	“azdev test iot --no-exitfirst”,
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ x ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ x ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ x ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).
